src/main/java/uk/gov/beis/els/util/StringUtils.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.beis.els.util;
+
+public class StringUtils {
+
+  private static final int DECIMAL_PLACES_DEFAULT = 0;
+
+  public static String toPercentage(float number) {
```

Comment:
Debated whether to have this has a string util or a number util combined with another string util...... settled on this.

Reply:
Nit but think it would be nicer to just move this into static methods in the `TemplatePopulator`  since this is only used in the context of template population. This also avoids needing to use the fully qualified path to `StringUtils` which otherwise hits the commons lang one.

Another nit, can we call this `decimalToPercentage` just to make it clear it only works with decimals